### ISSUE TITLE
Rebalancing

### ISSUE THREAD

**Issue by AVeryCoolSpider-PL** [Dong-Wo-Long/BetterDuStryMod#5]
You really need to buff Menders and Mend Projectors. And you have to make the Foundation core cost less resources - it can't be obtained without using Vaults.

**Comment by Dong-Wo-Long** [comment]
Have you tried testing Menders and Mend Projectors in heaps? In vanilla. If not, then try it, maybe it will give you the answer to the reason why I had to abandon them...

**Comment by Dong-Wo-Long** [comment]
As for the vaults, it seems to me that this is a very good challenge for the player. It forces you to twist yourself in some way in order to survive in a situation where you don't have the opportunity to improve the core, but you have the head on your shoulders to figure out how to solve this problem.

Nevertheless, I will consider your request a little later, but you still shouldn't count on much.

**Comment by AVeryCoolSpider-PL** [comment]
Well, i suppose you should add an alternative to menders - even if expensive, i would really appreciate to have something. The worst thing is that you made their healing power worse, but they still eat the same amount of power, which makes them useless. And i really, really hate that i can't get the big core without thorium. That makes the Foundation useless too, as after i have thorium i will just wait till i get the Nucleus. And units need rebelancing in my opinion - especially big groups of daggers. Come on, they are more powerful than the same amount of Novas, which are more expensive! I don't want to be rude, but i think that you focus too much on adding gimmicks rather than making it fun to play...

**Comment by Dong-Wo-Long** [comment]
Uploading Mindustry_TYJp6kphgy.mp4…

Maybe you're confused about something?

**Comment by AVeryCoolSpider-PL** [comment]
> Mindustry_TYJp6kphgy.mp4 
I can only say that in a situation where Novas and Daggers are in groups of the same count, Novas lose. Maybe this mod requires some other stuff to be downloaded, i don't know, but with it being on alone that is how the situation always ends

**Comment by Dong-Wo-Long** [comment]
Well, this situation is only possible probably when playing with an AI, but obviously not with a player. If you control Novas correctly, the probability that they will die from Daggers is about zero or less than 50%

**Comment by Dong-Wo-Long** [comment]
And I'm not even sure if the Nova's are inferior to the Daggers in any way, it's more like the Daggers are painful to watch - just now a group of 8 units were wiped off their face

**Comment by Dong-Wo-Long** [comment]
As for the problem with Vaults and Mender, I'll try to solve it today

**Comment by Dong-Wo-Long** [comment]
![Mindustry_fxccnDoWDD](https://github.com/user-attachments/assets/150ebba1-b3e3-4044-8429-fd9fe17b9727)

I really don't quite understand... That should be enough, right?

**Comment by Dong-Wo-Long** [comment]
![Mindustry_tkqusu6pc4](https://github.com/user-attachments/assets/3ae20eb9-6e52-4c7e-9789-d043abb140bc)

You don't even need Vaults to increase kernel capacity - you can't improve the kernel, but you can run it, right?

**Comment by AVeryCoolSpider-PL** [comment]
The thing is that you can't now put a Foundation on it. You would need to remove some of those containers, and that deletes resources

**Comment by Dong-Wo-Long** [comment]
Do you really feel bad about wasting resources, even though they are infinite...?

**Comment by AVeryCoolSpider-PL** [comment]
No... if you remove the containers, and the resources inside them, you don't have enough resources to build a bigger core

**Comment by Dong-Wo-Long** [comment]
![Mindustry_l2iQ6LeYKA](https://github.com/user-attachments/assets/78f63417-9aa9-449d-bb4a-cf7159815040)

So you mean something like this?

**Comment by AVeryCoolSpider-PL** [comment]
Yes. You used vaults, so there actually is enough place for resources to make a foundation - but containers have a much lower capacity. The Foundation in your mod takes 7000 titanium to be assembled, and only with vaults you are barely able to collect this much.

**Comment by Dong-Wo-Long** [comment]
I understand. But do you have to update the core immediately? Your progression should look like this:
1. First you use containers to run core foundation
2. Then you either do the same procedure with the Nucleus core as well, or go back and improve past cores with Vaults
3. Eventually in the middle of the game you will no longer have to go back to core Shard and Foundation, and you will forget about the existence of Vaults as a necessary measure to improve/run the core

**Comment by AVeryCoolSpider-PL** [comment]
This isn't a thing... using containers on a Shard core while keeping space for the Foundation core doesn't give you enough stored resources to build a Foundation core. It just isn't possible to get a Foundation without using thorium - go try that on a test map if you don't believe me. Just make sure you disable the infinite resources option.

**Comment by Dong-Wo-Long** [comment]
No, no, that's not what I meant!
You use containers (requiring titanium) to then run with core foundation
It makes sense that you can't upgrade core to Foundation with regular containers

**Comment by AVeryCoolSpider-PL** [comment]
There is no such a thing as "regular containers"
There are only Containers (unlocked after mining titanium) and Vaults (unlocked after mining thorium)

**Comment by AVeryCoolSpider-PL** [comment]
Just test it in a sandbox map, please. I am sure then you will understand my point of view. And just to be sure - i don't want to call you stupid in any way, i just think you don't realise what i exactly mean

**Comment by Dong-Wo-Long** [comment]
Geez, I know what you mean, but I don't want to make it so that you can just put Containers in and you can upgrade core to Foundation afterwards

**Comment by Dong-Wo-Long** [comment]
You should only have Containers at the beginning, with them you should NOT do core foundation, but instead just run the core on another sector, then already there, either you continue as well, or you learn Vaults, and then upgrade your old cores afterwards!!!

**Comment by AVeryCoolSpider-PL** [comment]
But... why?
If you unlock vaults, that means that you have thorium, and when you have thorium, you unlock a nucleus - which means the foundation is completely useless! Well, unless you made it so that the nucleus is also unreachable at it's tier...
(Quick explanation: there are copper tier, titanium tier, plastanium tier, thorium tier and surge tiers)

**Comment by AVeryCoolSpider-PL** [comment]
> You should only have Containers at the beginning, with them you should NOT do core foundation, but instead just run the core on another sector, then already there [...]

I think you don't know that you can't launch a foundation core if you have only a shard in the sector you are launching from.

**Comment by Dong-Wo-Long** [comment]
Foundation is an intermediate core, even if you somehow get thorium, will you be able to upgrade to Nucleus with the help of core Shard (which costs 25 thousand titanium, you can't have that much, 11 thousand at most)?

**Comment by Dong-Wo-Long** [comment]
> I think you don't know that you can't launch a foundation core if you have only a shard in the sector you are launching from.

Although, I think I've been told something like that...

**Comment by AVeryCoolSpider-PL** [comment]
Well, yes, but you don't do anything woth the Foundation - it is just another step on the road to nucleus, and after getting a nucleus, the foundation is useless

**Comment by Dong-Wo-Long** [comment]
Oh dear, I apologize, I seem to have forgotten about that

**Comment by AVeryCoolSpider-PL** [comment]
No problem, i tried modding and i onow how complticated it can be. Lowering the need of titanium for the Foundation should do the thing

**Comment by Dong-Wo-Long** [comment]
Since this is indeed a problem, it needs to be addressed

**Comment by AVeryCoolSpider-PL** [comment]
By "no problem" i wanted to show that in my opinion a reaction like "oh dear, i apologize" was... a bit too much, honestly. Well, that and the menders is everything, because the Nova-Dagger issue is a skill issue actually.

**Comment by Dong-Wo-Long** [comment]
Now the price of Foundation should not exceed 5500 resources

**Comment by AVeryCoolSpider-PL** [comment]
Thanks :> this is the l9ngest discussion i have ever had on Github

**Comment by Dong-Wo-Long** [comment]
With Mender I don't really know how to solve the problem, I'd probably describe it as “thin ice” that needs to be handled carefully

**Comment by Dong-Wo-Long** [comment]
> Thanks :> this is the longest discussion i have ever had on Github

I agree, it was an enjoyable pastime
